lib/src/Uncertainty/Algorithm/MetaModel/MetaModelValidation.cxx

```diff
@@ -128,7 +130,13 @@ Distribution MetaModelValidation::getResidualDistribution(const Bool smooth) con
   if (!isInitialized_) initialize();
   if (!smooth)
   {
-    return HistogramFactory().build(residual_);
+    const UnsignedInteger dimension = residual_.getDimension();
+    ComposedDistribution::DistributionCollection coll(dimension);
+    for (UnsignedInteger j = 0; j < dimension; ++ j)
+      coll[j] = HistogramFactory().build(residual_.getMarginal(j));
+    const Sample rankedSample((residual_.rank() + 0.5) / residual_.getSize());
+    Distribution copula(BernsteinCopulaFactory().build(rankedSample));
```

Comment:
A missing `const` here. BTW you don't need to transform the sample to [0,1]^d as it is done by BernsteinCopulaFactory for you, or at least when kFraction>1, because when kFraction it is not done, which is a bug...

Reply:
ok, fixing this too